Sasquatch/Config/Sasquatch ObjC.xcconfig

```diff
@@ -4,3 +4,5 @@ GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GCC_PREPROCESSOR_MACRO_SASQUATCH_OBJ
 CLANG_ENABLE_MODULES = NO
 OTHER_LDFLAGS = $(OTHER_LDFLAGS) -framework UserNotifications
 OTHER_LDFLAGS = $(OTHER_LDFLAGS) -framework SafariServices
+
+ARCHS[sdk=iphoneos*] = $(ARCHS_STANDARD) armv7s
```

Comment:
Let's make it standard only (it's supposed to be tested here). Also please add a comment for this and squash incorrect edits

```suggestion
ARCHS[sdk=iphoneos*] = $(ARCHS_STANDARD)
```